atomic_reactor/tasks/common.py

```diff
@@ -33,6 +33,7 @@ class TaskParams:
     namespace: str
     pipeline_run_name: str
     user_params: Dict[str, Any]
+    task_result: str
```

Comment:
```suggestion
    task_result: Optional[str]
```

Reply:
that didn't work, as binary params has mandatory platforms